src/services/platform.services.js

```diff
@@ -0,0 +1,39 @@
+import * as childProcess from 'child_process';
+import * as os from 'os';
+import * as path from 'path';
+import { remote } from 'electron';
+import { PACKAGE_MANAGER } from '../config/app';
+
+// Returns true if the OS is Windows
+export const isWin = (): boolean => /^win/.test(os.platform());
```

Comment:
this probably should be a constant instead of a function, there is no way the platform can change during an execution